package.json

```diff
@@ -52,11 +52,5 @@
     "simple-git-hooks": {
         "pre-commit": "npx lint-staged",
         "commit-msg": "npx commitlint --edit $1"
-    },
-    "dependencies": {
-        "@fortawesome/fontawesome-svg-core": "6.3.0",
```

Comment:
Is it good that "@fortawesome/fontawesome-svg-core": "6.3.0" is missing in frontend/package.json?

Reply:
it already exist in 19 line of frontend/package.json